Files/Filesystem/INavigationControlItem.cs

```diff
@@ -11,6 +11,7 @@ public interface INavigationControlItem
         public string HoverDisplayText { get; }
 
         public NavigationControlItemType ItemType { get; }
+        public System.Collections.Generic.List<INavigationControlItem> SideBarChildItems { get; }
```

Comment:
Should this go here or under LocationItem instead? Will non-location items ever need child items?

Also a nit: Rename to ChildItems

Reply:
Also the list will likely need to be Observable

Reply:
> 
> 
> Should this go here or under LocationItem only? Will non-location items ever need child items?
> 
> Also a nit: Rename to ChildItems

Hi, @duke7553 i already submit your requests




> 
> 
> Also the list will likely need to be an ObservableCollection

@gave92 i already replace by type by ObservableCollection



> 
> 
> @oleitao Is it possible to make the favorites section collapsible as well?

@gave92  and @yaichenbaumi added collapse favorites section